timemachine/cpp/src/potential.cu

```diff
@@ -22,8 +23,6 @@ void Potential::execute_host(
     double *d_p;
     double *d_box;
 
-    const int D = 3;
```

Comment:
took a small step toward DRYing these definitions (moved to a `static const` the header)